### PR TITLE
Issue #1348 Abstract the code to create progress dots

### LINKED ISSUE #1348: Abstract the code to create progress dots
As per the fix of #1277 in the PR #1340 , we missed extracting the code to create progress dots here https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/start.go#L300.

Probably, good to extract this into `pkg/util/utils.go` along with functions like `StartProgressDots`.

### PR DESCRIPTION
**Pull request by budhrg**
Fix #1348

### CODE REVIEW

**Comment by gbraad** [comment]
Had placed the code on the wrong pull request, so here goes again:

Move to subdir of pkg/util

```go
package progress

// Returns a new instance of the ProgressDots with a given interval
func New(interval int) *ProgressDots{
	return &ProgressDots{
            ...
	}
}

type ProgressDots struct {
    interval int
}

func (s *ProgressDots) Start() {
}

func (s *ProgressDots) Stop() {
}
```

File: cmd/minishift/cmd/start.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ import (
 
 	"github.com/minishift/minishift/pkg/util"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/minishift/minishift/pkg/util/progress"
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start.go]: The package name `progress` should be changed to `progressDots` or something better.

Comment (gbraad) [reply cmd/minishift/cmd/start.go]: package names do not have casing progressdots is possible, but not sure if this really adds value.

Comment (gbraad) [reply cmd/minishift/cmd/start.go]: If progressDots is used as a package name, it clashes with:
```
progressDots := progress.New()
```

and therefore would need an override from the import to use
```
    progressdots"github.com/minishift/minishift/pkg/util/progressDots"
```

`progressdots` it is... changing and merging...

**Comment by gbraad** [comment]
Merged as https://github.com/minishift/minishift/commit/9aa240de4dff47619b3823a96291553fd46b5a21

**Pull request closed without merging**